reana_db/models.py

```diff
@@ -150,22 +155,21 @@ def __init__(self,
             name=name,
             owner_id=owner_id).\
             order_by(Workflow.run_number.desc()).first()
-        if not last_workflow:
-            self.run_number = 1
+        if restart:
```

Comment:
An observation regarding how the run number is exposed to users. If you run a workflow once, you will see:
```console
$ rc run 
[INFO] Creating a workflow...
workflow.1.0
[INFO] Uploading files...
File code/helloworld.py was successfully uploaded.
File data/names.txt was successfully uploaded.
[INFO] Starting workflow...
workflow.1.0 is running
```
It seems a bit weird to see the `1.0` when I haven't used the restart feature. I would rather expect to see ` 1` and then when I run `reana-client start --restart -w workflow.1` I would expect to see `1.1`.

I think we could easily do this here by using:
```python
In [1]: run_number_with_restart = 1.1 

In [2]: run_number_with_no_restart = 2.0                                    

In [3]: run_numbers = [run_number_with_restart, run_number_with_no_restart] 

In [4]: for run_number in run_numbers:
   ...:     if run_number.is_integer(): 
   ...:         print(int(run_number))
   ...:     else: 
   ...:         print(run_number) 
   ...:            
1.1
2
```

If we do it, I would try to move the whole `run_number_increase` logic to a different method, so we keep the constructor tidy.

Reply:
Thanks for suggestions, it was addressed.